package.json

```diff
@@ -66,14 +66,16 @@
     "build": "tsc",
     "test": "cross-env CI=true jest --coverage",
     "test:watch": "jest --watch",
-    "docs": "npm run docs:markdown & npm run docs:readme-toc",
+    "docs": "npm run docs:markdown & npm run generate:readme-toc",
     "docs:markdown": "./node_modules/.bin/jsdoc2md lib/index.js -f lib/**/*.js > API.md",
-    "docs:readme-toc": "markdown-toc -i README.md",
     "lint": "eslint --max-warnings 0 --config .eslintrc .",
     "lint:fix": "eslint --max-warnings 0 --config .eslintrc . --fix",
-    "get-version": "echo $npm_package_version",
-    "get-name": "echo $npm_package_name",
-    "release": "semantic-release"
+    "get:version": "echo $npm_package_version",
+    "get:name": "echo $npm_package_name",
+    "release": "semantic-release",
+    "generate:assets": "npm run docs",
+    "generate:readme-toc": "markdown-toc -i README.md",
```

Comment:
```suggestion
    "generate:readme:toc": "markdown-toc -i README.md",
```

Reply:
@fmvilas you've got the eye of the tiger man!

https://youtu.be/CevxZvSJLk8?t=66

Reply:
fix pushed, approve 🙏🏼